apps/examples/e2e/tests/test-mobile-style-panel.spec.ts

```diff
@@ -0,0 +1,24 @@
+import test, { expect } from '@playwright/test'
+import { setup } from '../shared-e2e'
+
+test.describe('mobile ui', () => {
+	test.beforeEach(setup)
+	test('style panel opens and closes as expected', async ({ isMobile, page }) => {
+		test.skip(!isMobile, 'only run on mobile')
+		const mobileStylesButton = page.getByTestId('mobile-styles.button')
+		const stylePanel = page.getByTestId('style.panel')
```

Comment:
How about we introduce `getElement` on the `stylePanel` fixture (and `toolbar` if needed), which just returns `page.getByTestId('style.panel')`?